FWCore/Services/plugins/InitRootHandlers.cc

```diff
@@ -875,6 +875,10 @@ namespace edm {
       if(debugLevel >0) {
 	gDebug = debugLevel;
       }
+
+      // Enable Root implicit multi-threading
+      bool imt = pset.getUntrackedParameter<bool>("EnableIMT");
+      if (imt) ROOT::EnableImplicitMT(4);
```

Comment:
Its a placeholder argument. It has no effect since the tbb system is already initialized.